enterprise/coderd/prebuilds/metricscollector_test.go

```diff
@@ -307,6 +282,127 @@ func TestMetricsCollector(t *testing.T) {
 	}
 }
 
+func TestMetricsCollector_DuplicateTemplateNames(t *testing.T) {
+	t.Parallel()
+
+	if !dbtestutil.WillUsePostgres() {
+		t.Skip("this test requires postgres")
```

Comment:
why does this test require postgres?

Reply:
Doesn't seem so. Fixed in ceb64a0b5.

Reply:
Hhmm reverted, apparently without it we get incorrect provisioner jobs being picked up:
https://github.com/coder/coder/actions/runs/15040106313/job/42269672510?pr=17839
I don't have time to look into that now, and this fixes it; let's look into it another time.